-- cmake_modules/INSTALL_CMAKE_MODULESDIR.cmake
-- cmake_modules/INSTALL_CMAKE_MODULESDIR.cmake
@@ -10,6 +10,8 @@
 IF(NOT INSTALL_CMAKE_MODULESDIR)
 
   FILE(GLOB POSSIBLE_PATHS /usr/share/cmake*)
+  LIST(SORT POSSIBLE_PATHS)
+  LIST(REVERSE POSSIBLE_PATHS)
 
   FIND_PATH(INSTALL_CMAKE_MODULESDIR Modules
     PATHS
